cuda/components/warp_blas.cuh

```diff
@@ -327,13 +327,13 @@ __device__ __forceinline__ void multiply_transposed_vec(
  */
 template <
     int max_problem_size, typename Group, typename MatrixValueType,
-    typename VectorValueType,
+    typename VectorValueType, typename Closure,
     typename = xstd::enable_if_t<group::is_communicator_group<Group>::value>>
 __device__ __forceinline__ void multiply_vec(
     const Group &__restrict__ group, uint32 problem_size,
     const VectorValueType &__restrict__ vec,
     const MatrixValueType *__restrict__ mtx_row, uint32 mtx_increment,
-    VectorValueType *__restrict__ res, uint32 res_increment)
+    VectorValueType *__restrict__ res, uint32 res_increment, Closure closure_op)
```

Comment:
Could you also document this Closure in the doxygen above? Something like
```
@tparam Closure  type of the function used as to write the result
[...]
@param closure_op  Operation that is performed when writing to
                   `res[group.thread_rank() * res_increment]` as 
                   `closure_op(res[group.thread_rank() * res_increment], out)`
                   where `out` is the result of \f$ res = mtx \cdot vec \f$.
```
or similar.

Reply:
Oops, thanks for pointing that out. This is fixed.